src/BlockEntities/NoteEntity.h

```diff
@@ -56,6 +56,10 @@ class cNoteEntity :
 	virtual bool UsedBy(cPlayer * a_Player) override;
 	virtual void SendTo(cClientHandle &) override {}
 
+	/** Calculates the pitch to be send to the players
+	from a pitch value from 0 to 24 as incremented by the IncrementPitch function */
+	static float GetPitch(char a_Note);
```

Comment:
Perhaps also rename to `PitchFromNote` to be absolutely clear about this?

Reply:
Thats a nice name

Reply:
Renamed